wallet/src/main/java/com/android/identity_credential/wallet/credman/CredmanPresentationActivity.kt

```diff
@@ -63,7 +63,7 @@ class CredmanPresentationActivity : FragmentActivity() {
 
     private fun addDeviceNamespaces(
         documentGenerator: DocumentGenerator,
-        authKey: Credential,
+        authKey: MdocCredential,
```

Comment:
Would be nice to rename `authKey` -> `credential` in place where you make changes anyway (also elsewhere, use e.g. `git show` and search)